mewarpx/mewarpx/mwxrun.py

```diff
@@ -250,5 +251,22 @@ def set_phi_grid(self, phi_data):
                 raise
         _libwarpx.set_phi_grid_fp(self.lev)
 
+    def eval_expression_t(self, expr):
+        """Function to evaluate an expression that depends on time, at the
+        current simulation time using the WarpX parser.
+
+        Arguments:
+            expr (str or float): Expression to evaluate.
+
+        Returns:
+            (float) Value of the expression at the current simulation time.
+        """
+        if isinstance(expr, float):
+            return expr
+        else:
+            return _libwarpx.libwarpx.eval_expression_t(
+                ctypes.c_char_p(expr.encode('utf-8'))
```

Comment:
This line is throwing an attribute error in circle ci, when running the test_circle_emitter.py

Reply:
```
        if isinstance(expr, str):
            return _libwarpx.libwarpx.eval_expression_t(
                ctypes.c_char_p(expr.encode('utf-8'))
        else:
            return expr
```